feed.rss

```diff
@@ -1388,6 +1388,40 @@
 				<li><a href="https://edu.leankanban.com/users/david-anderson">David Anderson</a></li>
 				<li><a href="https://en.wikipedia.org/wiki/Kaizen">Kaizen</a></li>
 				</ul>]]></itunes:summary>
+    </item>
+	<item>
+      <guid>https://artemzin.com/static/thecontext/episodes/The.Context.episode.20.part2.mp3</guid>
+      <title>Episode 20, Part 2: public final Agile</title>
+      <link>https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_21.md</link>
+      <atom:link href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/97" rel="replies" type="text/html" />
+      <itunes:duration>01:03:12</itunes:duration>
+      <pubDate>Wed, 1 August 2018 09:36:00 EST</pubDate>
+      <enclosure length="60920969" type="audio/mpeg" url="https://artemzin.com/static/thecontext/episodes/The.Context.episode.21.mp3" />
+      <itunes:summary><![CDATA[<h1>Episode 21: Rx Must Die</h1>
+
+				<h3><a href="https://github.com/artem-zinnatullin/TheContext-Podcast">How to listen &amp; subscribe</a></h3>
+
+				<ul>
+				<li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/99">Discussion after the episode</a></li>
+				</ul>
+
+				<p>This time we have a special episode: we have recorded a panel discussion at droidcon Berlin 2018 (June 25) with the name Rx Must Die.</p>
```

Comment:
Maybe reword to say that we collaborated with Droidcon Berlin and they provided us a recording?

Reply:
hm, well , can you edit it ... I run out of good ideas how to write it down